permissions/plugin-metrics.yml

```diff
@@ -6,6 +6,7 @@ issues:
 paths:
 - "org/jenkins-ci/plugins/metrics"
 developers:
+- "@cloudbees-developers"
```

Comment:
already present on line 12/13 below :)

```suggestion
```

Reply:
removed in addd22728a8accbc679041d3116d0925cf72d248